eng/source-build-patches/0011-Fix-a-couple-issues.patch

```diff
@@ -0,0 +1,96 @@
+From 594c716ad8993df91b4cbade593b9bee7cacae47 Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Thu, 3 Sep 2020 06:29:25 -0500
+Subject: [PATCH 11/14] Fix a couple issues.
+
+---
+ AspNetCore.sln                                |  2 -
+ src/Components/Components.slnf                |  1 -
+ src/Components/ComponentsNoDeps.slnf          |  1 -
+ .../Driver/Wasm.Performance.Driver.csproj     | 37 -------------------
+ 4 files changed, 41 deletions(-)
+ delete mode 100644 src/Components/benchmarkapps/Wasm.Performance/Driver/Wasm.Performance.Driver.csproj
```

Comment:
If this isn't automatically excluded from source builds, please let us know the concern. May just need to disable a couple of targets in the project.

Reply:
It isn't automatically excluded.  I agree that it should be easy to disable.